theme/templates/newbase.html

```diff
@@ -79,7 +79,10 @@
             display: block;
         }
     </style>
-    <script src="{% static 'mezzanine/js/jquery-1.8.3.min.js' %}"></script>
+{# Pabitra: commenting the use of mezzanine supplied jquery js file as the new version of mezzaine no more uses jquery 1.8.3 #}
+{#    <script src="{% static 'mezzanine/js/jquery-1.8.3.min.js' %}"></script>#}
+{# Pabitra: using jquery 1.8.3 - this is the version used by mezzanine before the mezzanine upgrade to 5.x  and this vesrion of jquery works with our existing csutome JS code #}
```

Comment:
not sure if you want to retain these comments

Reply:
removed comments.